app/build.gradle

```diff
@@ -200,6 +201,9 @@ dependencies {
     implementation 'org.jsoup:jsoup:1.12.1'
     implementation 'com.github.chrisbanes:PhotoView:2.3.0'
     implementation "com.github.skydoves:balloon:1.2.1"
+    implementation 'com.google.firebase:firebase-ml-natural-language:22.0.1'
```

Comment:
It looks like language detection is no longer done through Firebase (it's deprecated), but now done directly with ML Kit. https://developers.google.com/ml-kit/language/identification/android